articles/azure-developer-cli/reference.md

```diff
@@ -1,278 +1,324 @@
+* $HOME/.azd on Linux and MacOS
+* %USERPROFILE%\.azd on Windows
 
-Example:
+The configuration directory can be overridden by specifying a path in the AZD_CONFIG_DIR environment variable to the default.
 
-```azdeveloper
-azd config set defaults.subscription <yourSubscriptionID>
-azd config set defaults.location eastus
 ```
- 
+azd config reset [flags]
+```
+
 ### Options
 
-```azdeveloper
--h, --help   Gets help for set.
+```
+  -h, --help   Gets help for reset.
 ```
 
 ### Options inherited from parent commands
 
-```azdeveloper
--C, --cwd string           Sets the current working directory.
-    --debug                Enables debugging and diagnostics logging.
--e, --environment string   The name of the environment to use.
-    --no-prompt            Accepts the default value instead of prompting, or it fails if there is no default.
+```
+  -C, --cwd string   Sets the current working directory.
+      --debug        Enables debugging and diagnostics logging.
+      --no-prompt    Accepts the default value instead of prompting, or it fails if there is no default.
 ```
 
 ### See also
 
-- [azd config](#azd-config): Manage user configurations
-- [Back to top](#azd)
+* [azd config](#azd-config): Manage Azure Developer CLI configuration
+* [Back to top](#azd)
 
-## azd config reset
+## azd config set
 
-Resets all configuration in $AZURE_CONFIG_DIR/config.json to the default.
+Sets a configuration
 
-```azdeveloper
-azd config reset [flags]
+### Synopsis
+
+Sets a configuration in the configuration path. 
+
+The default value of the config directory is: 
+* $HOME/.azd on Linux and MacOS
+* %USERPROFILE%\.azd on Windows
+
+The configuration directory can be overridden by specifying a path in the AZD_CONFIG_DIR environment variable.
+
+```
+azd config set <path> <value> [flags]
+```
+
+### Examples
+
+```
+$ azd config set defaults.subscription <yourSubscriptionID>
+$ azd config set defaults.location eastus
 ```
 
 ### Options
 
-```azdeveloper
--h, --help   Gets help for reset.
+```
+  -h, --help   Gets help for set.
 ```
 
 ### Options inherited from parent commands
 
-```azdeveloper
--C, --cwd string           Sets the current working directory.
-    --debug                Enables debugging and diagnostics logging.
--e, --environment string   The name of the environment to use.
-    --no-prompt            Accepts the default value instead of prompting, or it fails if there is no default.
+```
+  -C, --cwd string   Sets the current working directory.
+      --debug        Enables debugging and diagnostics logging.
+      --no-prompt    Accepts the default value instead of prompting, or it fails if there is no default.
 ```
 
 ### See also
 
-- [azd config](#azd-config): Manage user configurations
-- [Back to top](#azd)
+* [azd config](#azd-config): Manage Azure Developer CLI configuration
+* [Back to top](#azd)
 
 ## azd config unset
 
-Removes a configuration in $AZURE_CONFIG_DIR/config.json. For example:
+Unsets a configuration
```

Comment:
Keep previous version - "Removes a configuration in $AZURE_CONFIG_DIR/config.json."